src/ui/components/BoxScore.football.tsx

```diff
@@ -84,7 +85,7 @@ export const sortByStats = (
 ) => {
 	return (a: any, b: any) => {
 		for (const [index, order] of sortBys) {
-			const stat = stats[index];
+			const stat = index.includes(":") ? index.split(":")[1] : index;
```

Comment:
Is this really needed? Aren't all the sortable columns here stats, just like the hockey box score?

Reply:
removed the double click on hockey and football

Reply:
Explained on discord about changes made to `updateSortBys`

Reply:
I don't understand how that affects this line of code.

Reply:
so because the columns are sortable a SortBy would be something like ["stat:pssCmp", "desc"].
Leaving it as `stats[index]` wouldn't work because "stat:pssCmp" is not a index. We can get the stat name (in this example "pssCmp") by parsing "stat:pssCmp".

Reply:
Nah I was being stupid, my bad